avg_temp_job.txt

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+#SBATCH -A IHESD
+#SBATCH -t 05:00:00
+#SBATCH -N 1
+
+#SBATCH -n 1
+#SBATCH -J averageAnnualTemps
+
```

Comment:
please add the line `source /etc/profile.d/modules.sh >& /dev/null` before you call module load. Recently I've run into some issues submitting sbatch scripts without it.